controller_manager/test/test_spawner_unspawner.cpp

```diff
@@ -274,3 +274,13 @@ TEST_F(TestLoadController, unload_on_kill)
 
   ASSERT_EQ(cm_->get_loaded_controllers().size(), 0ul);
 }
+
+TEST_F(TestLoadController, spawner_test_load_only_in_arg)
+{
+  EXPECT_EQ(call_spawner("ctrl_2 -c test_controller_manager --load-only "), 0);
```

Comment:
```suggestion
  cm_->set_parameter(rclcpp::Parameter("ctrl_2.type", test_controller::TEST_CONTROLLER_CLASS_NAME));
  EXPECT_EQ(call_spawner("ctrl_2 -c test_controller_manager --load-only "), 0);
```

You can either do this or pass an argument to the spawner